hawkbit/src/ddi/deployment_base.rs

```diff
@@ -92,7 +93,7 @@ struct ChunkInternal {
     artifacts: Vec<ArtifactInternal>,
 }
 
-#[derive(Debug, Deserialize)]
+#[derive(Clone, Debug, Deserialize)]
```

Comment:
You don't need `Clone` here as you manually clone the key and the value.

Reply:
Forgot to remove the unneeded `Clone` from previous tests.
Removed it